test/kubemark/resources/hollow-node_template.yaml

```diff
@@ -56,7 +56,7 @@ spec:
         command:
         - /bin/sh
         - -c
-        - ./kubemark.sh --morph=kubelet --name=$(NODE_NAME) --kubeconfig=/kubeconfig/kubelet.kubeconfig $(CONTENT_TYPE) {{kubelet_verbosity_level}} 1>>/var/log/kubelet-$(NODE_NAME).log 2>&1
+        - ./kubemark.sh --morph=kubelet --name=$(NODE_NAME) --kubeconfig=/kubeconfig/kubelet.kubeconfig $(CONTENT_TYPE) --alsologtostderr {{kubelet_verbosity_level}} 1>>/var/log/kubelet-$(NODE_NAME).log 2>&1
```

Comment:
I guess it would only be better to have multiple places where the logs are stored (in the case of --alsologtostderr we would have the logs available also through docker). WDYT?